### PR TITLE
Use the validation data for validation, and the test data for testing

### PR DESCRIPTION
**Pull request by bh9**
Previously, the test file was used for both testing and validation. This is probably because the testing valiables were valid_* and the validation variables were test_*, which no doubt led to some confusion. Fix to use validation data to validate, and test data to test, and rename variables to match their purpose

### CODE REVIEW

**Comment by bh9** [comment]
Need to update the notebooks to match the renaming

**Review by rschwess — APPROVED** [review]
Yeah, for some reason I got taught test set for testing during/after training and validation set as the hold out for the very end. But as the rest of the ML field is doing it the other way around I shuld finally yield to that convention :)
